frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -166,7 +166,7 @@ export const createOrUpdateDeployment = (
 
   const imageRef =
     registry === RegistryType.External
-      ? `${imgName || name}:${tag}`
+      ? `${imgName || name}:${tag || imageStreamTag}`
```

Comment:
We can directly use `tag` in this case `imageStreamTag` from `isi` since it is set for both external and internal registry scenarios.
Also when `registry` is `external` then `imgName`  and `tag` which is coming from `imageStream` will always be empty since it is set only when `registry` is internal.

 ```suggestion
      ? `${name}:${imageStreamTag}`
```